_includes/manuals/3.2/1.2_release_notes.md

```diff
@@ -2,12 +2,24 @@
 
 ### Headline features
 
+#### Host reports
```

Comment:
I don't think this should be a headline feature if there is no installer support. I also think we shouldn't introduce headline features in a bugfix release.

IMHO we should either postpone the release (and make sure everything gets in) if we want to make it a headline feature or we release it in 3.2.1 as an optional new feature. I'm leaning to the latter.

Reply:
Ok how about Preview feature?

Reply:
I moved it down and marked them as a preview feature.

Reply:
I think the "preview" makes sense if we provide instructions how to install. We want to let users know they can start playing with that. Also that prepares the ground for the release with 3.3.